src/components/TutorialManager.tsx

```
@@ -35,6 +43,31 @@ export const TutorialManager = () => {
     t
   } = useTutorial()
   const hasContent = isVisible && content !== null
+  const panelRef = useRef<HTMLDivElement>(null)
+
+  // Publish the panel's height as `--tutorial-inset` so bottom-anchored scenes
+  // can reserve room for it. Narrow viewports have no space to place the panel
+  // clear of the scene's own controls, and the app body is `overflow: hidden`,
+  // so without this the panel sits on top of buttons that can then never be
+  // tapped. Consumed by `MainMenu` under `max-sm`.
+  useLayoutEffect(() => {
```

Biome (2.5.7)

[error] This hook specifies more dependencies than necessary: hasContent.

(lint/correctness/useExhaustiveDependencies)